scrapy/downloadermiddlewares/retry.py

```diff
@@ -147,6 +138,16 @@ def __init__(self, settings):
         )
         self.priority_adjust = settings.getint("RETRY_PRIORITY_ADJUST")
 
+        if not hasattr(
+            self, "EXCEPTIONS_TO_RETRY"
+        ):  # If EXCEPTIONS_TO_RETRY is not "overriden"
+            self.exceptions_to_retry = tuple(
+                load_object(x) if isinstance(x, six.string_types) else x
```

Comment:
```suggestion
                load_object(x) if isinstance(x, str) else x
```

Reply:
Actually in both these lines just `load_object(x)` should work, as `load_object()` works with any classes and just returns them.